KQ7365/orgamate-client
# See PR comment

## I created functionality within the `Organize` component that allows a user to add and delete a new item: 

## Steps To Test
1. ensure you have the debugger running from the server. If not, pull down orgamate-server git: `kq-liveServerReady` branch, run the virtual environment, and start the debugger.
2. in a separate terminal now, pull down and step into the `kq-organizeItemInputFormBuild` branch and run the cmd `npm run dev' to access the web browser
3. Open the dev tools and monitor the Console to ensure no fatal errors throughout this test.
4. If you do not have a login, register a user and then attempt login.
5. Once logged in navigate to the `Organize` page via the Navbar
6. Add a new item by filling out the following fields: Image URL, Item name, Description, Select a category, Select a location, and select multiple tags and click Add New Item
7. Ensure you see the item immediately added to the page
8. To test the delete, click the delete button on an individual item and ensure it is immediately removed.
9. Lastly and once again, ensure no fatal errors located within the console.
